account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -169,10 +169,16 @@ def create_company(self):
     def taxes_by_company(self, company_id, match_tax_ids):
         AccountTax = self.env['account.tax'].sudo()
         account_taxes = AccountTax.browse(match_tax_ids)
-        return AccountTax.search([
-            ('description', 'in', account_taxes.mapped('description')),
-            ('company_id', '=', company_id),
-        ]).ids
+        xml_ids = account_taxes.get_external_id().values()
+        # If any tax doesn't have xml, we won't be able to match it
+        if "" in xml_ids:
+            return []
+        records = []
+        for xml_id in xml_ids:
+            module, ref = xml_id.split(".", 1)
+            company, name = ref.split("_", 1)
+            records.append(self.env.ref("%s.%s_%s" % (module, company_id, name)).id)
```

Comment:
The XML-ID may not exists:
```suggestion
            record = self.env.ref("%s.%s_%s" % (module, company_id, name), False)
            if record:
                records.append(record.id)
```

Reply:
Done